install/kubernetes/cilium/values.yaml

```diff
@@ -1410,7 +1410,7 @@ hubble:
       #
       #   --set hubble.redact.enabled="true"
       #   --set hubble.redact.kafka.apiKey="true"
-      apiKey: false
+      apiKey: true
```

Comment:
can you update values.yaml.tmpl file instead, and then run `make -C install/kubernetes` to regenerate this file?